ctk/common/src/test/kotlin/org/codice/compilance/ReportListener.kt

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright (c) 2019 Codice Foundation
+
+Released under the GNU Lesser General Public License; see
+http://www.gnu.org/licenses/lgpl.html
+*/
+package org.codice.compilance
+
+import io.kotlintest.Description
+import io.kotlintest.extensions.TestListener
+import org.codice.compliance.utils.report.Report
+
+/**
+ * Listener used to reset the Report map for testing purposes before each test
+ */
+object ReportListener : TestListener {
+
+    override fun beforeTest(description: Description) {
```

Comment:
✏️ The first test wouldn't need this beforehand. Technically, you could avoid the extra reset by doing `afterTest` instead.